miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppManifestVerifier.kt

```diff
@@ -0,0 +1,72 @@
+package com.rakuten.tech.mobile.miniapp
```

Comment:
It seems like the root package isn't the right place for this - usually most classes in the root are public classes. Should this go under `storage` instead?